torch/_inductor/scheduler.py

```diff
@@ -2357,9 +2360,20 @@ def add_user(
                 if s not in unbacked_symbol_to_origin_node:
                     unbacked_symbol_to_origin_node[s] = node.get_name()
 
-            unbacked_symbol_uses = sorted(
-                node.node.get_free_symbol_uses(unbacked_only=True), key=lambda x: x.name
-            )
+            unbacked_symbol_uses = node.node.get_free_symbol_uses(unbacked_only=True)
+
+            for read in node.read_writes.reads:
+                if isinstance(read, MemoryDep):
+                    unbacked_symbol_uses = unbacked_symbol_uses.union(
+                        free_unbacked_symbols(read.index)
+                    )
+            for write in node.read_writes.writes:
+                if isinstance(write, MemoryDep):
+                    unbacked_symbol_uses = unbacked_symbol_uses.union(
+                        free_unbacked_symbols(write.index)
+                    )
```

Comment:
What's going on here?

Reply:
We have a node of type ComputedBuffer such that an unbacked symbols appears in node.get_read_writes() but not in    get_free_symbol_uses() I will move this logic inside     def get_free_symbol_uses() its better there.